CODEOWNERS

```diff
@@ -4,14 +4,54 @@
 # It uses the same pattern rule for gitignore file
 # https://git-scm.com/docs/gitignore#_pattern_format
 
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# the following owners will be requested for
+# review when someone opens a pull request.
+* @nkuchta
+
 # Azure Monitor Agent Extension
 /AzureMonitorAgent/ @Azure/geneva-linux-agents
 
+# CustomScript Extension
+/CustomScript/ @D1v38om83r @nkuchta @vrdmr
+
 # Diagnostics (LAD) Extension
 /Diagnostics/ @Azure/geneva-linux-agents
 
+# DSCForLinux Extension
+/DSC/ @nitinbps @kwirkykat
+
 # OMS Agent Extension
 /OmsAgent/ @Azure/geneva-linux-agents
 
+# OpenCensus Translator Extension
+/opencensus-service/ @Azure/geneva-linux-agents
+
 # VMAccess Extension
 /VMAccess/ @D1v38om83r @nkuchta @vrdmr
```

Comment:
@D1v38om83r, @nkuchta  - Could you please create a Team in Azure for extensions stuff as well? It'll become easier to manage the members.